src/ex1/script.js

```diff
@@ -0,0 +1,165 @@
+const allTodosList = document.getElementById("all-todos-list");
+
+const unsorted = 0;
+const sortedAsc = 1;
+const sortedDesc = 2;
+let isListSorted = unsorted;
+
+const todosArray = ['Wash the dishes', 'Walk the dog', 'Water the flower', 'Feed the baby'];
+const inputTitle = document.getElementById("new-todo-title");
+const amountOfTodosInfo = document.getElementById("amount-info");
+
+const clearAllButton = document.getElementById("clear-all-button");
+const addTodoForm = document.getElementById("add-todo");
+const sortListButton = document.getElementById("sort-list-button");
+
+clearAllButton.addEventListener('click', onClearAllButtonClicked);
+addTodoForm.addEventListener('submit', onAddTodoFormSubmitted);
+sortListButton.addEventListener('click', onSortListButtonClicked);
+
+todosArray.forEach(todoText => {
+  addTodoItem(todoText, false);
+  displayFooterAndImage();
```

Comment:
why is this called for every item inside the items list?
Also it's confusing as the naming says "display", but actually, it's "display OR hide".
Also, the term image is a bit confusing and coupled to the current solution, having a more descriptive name like "showOrHideEmptyState" might make this more readable

Reply:
fixed